Formula/texlive-bin.rb

```diff
@@ -1,4 +1,4 @@
-class Texlive < Formula
+class TexliveBin < Formula
```

Comment:
We should consider keeping the name and just adding `caveats` as with other formulae.